Gruntfile.js

```diff
@@ -6,12 +6,14 @@ camelcase: ["error", {"properties": "never"}]
 module.exports = function (grunt) {
   'use strict';
 
+  grunt.loadNpmTasks('grunt-exec');
```

Comment:
At some point, we're going to have to stop adding tasks to Grunt if we're ever going to get rid of it.

Should now be that time?

Reply:
Since we want this to be part of the build it needs to go in the grunt `build` step so `npm develop` builds it as well. We won't be moving off grunt any time soon and not in this PR so a grunt step is fine for now.

Reply:
I don't think I said it needed to be in grunt :P. We did a good while back agree not to add more grunt tasks if we could at all avoid it. It seems viable to put this work in prebuild / predevelop scripts instead. Did you try that @gaiety-deque? If that was more difficult I can live with a Grunt solution, but if we can avoid these extra dependencies + more grunt tasks I think that's the way to go on this.

Reply:
Apologies @WilcoFiers I didn't mean to imply that you explicitly said it had to be in the _grunt_ build step, just that the issue requested it be a part of the build step. Since our build step just calls "grunt" currently and the pre/post build steps were more test-like in nature it felt like we were intentionally doing these things in grunt.

Made a change [1785f2c](https://github.com/dequelabs/axe-core/pull/4456/commits/1785f2c7b5845cf1611051586960a4efb271937c) to move it to be commands run in the package json scripts.